tests/integration/pkg/systemdnode/systemnode.go

```diff
@@ -25,7 +24,31 @@ func New(clients *clients.Clients, namespace, script string) (*corev1.Pod, error
 		return nil, err
 	}
 
-	imagesPath := fmt.Sprintf("/var/lib/rancher/%s/agent/images", os.Getenv("DIST"))
+	// We are providing the following files/configmap in order to force K3s/RKE2 to use the cgroupfs cgroup driver,
+	// rather than systemd.
+	// We must also drop-in replace the type of service for systemd as we are clearing the notify socket which would
+	// normally cause systemd to hang waiting for the unit to activate. Eventually, when
+	// https://github.com/rancher/rke2/issues/3240 is resolved, we should be able to roll back this workaround. If the
```

Comment:
This should also fix the issues I was having running the tests on my linux dev box.